tests/testthat/test-internals.R

```diff
@@ -1,14 +1,14 @@
-context("internals")
-
 test_that(".quick_fetch() success case", {
-  options('meetupr.use_oauth' = FALSE)
+  withr::local_options(list(meetupr.use_oauth = FALSE))
+  set_api_key("yay")
+
   res <- with_mock(
     `httr::GET` = function(url, query, ...) {
-      load(here::here("tests/testdata/httr_get_find_groups.rda"))
+      load(test_path("testdata/httr_get_find_groups.rda"))
       return(req)
     },
-    res <- .quick_fetch(api_url = "fake url", # intentionally invalid as there is currently no validation
-                      api_key = "I <3 R-Ladies")
+    # intentionally invalid as there is currently no validation
+    res <- .quick_fetch(api_url = "fake url")
```

Comment:
This test didn't really seem to be *about* the API key? So I just set it above, like all the other tests, and now tests pass for me, without needing to set an env var.